ASPNetCoreMasters/Startup.cs

```diff
@@ -26,6 +28,11 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddControllers();
+            services.AddScoped<IItemService, ItemService>();
+            services.AddScoped<IItemRepository, ItemRepository>();
+            services.AddScoped<DataContext>();
```

Comment:
i recommend using Singleton so you will be able to test the feature requirements easier